pkgs/top-level/all-packages.nix

```diff
@@ -25942,20 +25942,20 @@ with pkgs;
     postgresql_15_jit
     postgresql_16_jit
   ;
-  postgresql = postgresql_15;
-  postgresql_jit = postgresql_15_jit;
+  postgresql = postgresql_16;
+  postgresql_jit = postgresql_16_jit;
```

Comment:
Sounds good, I'd suggest we do the policy change on the next postgresql release (which will include a v17 iirc).
Then, it's explicit because the default server version is not the default package anymore.